sonaranalyzer-dotnet/rspec/vbnet/S4507_vb.net.html

```diff
@@ -22,29 +22,38 @@ <h2>Recommended Secure Coding Practices</h2>
 <h2>Sensitive Code Example</h2>
 <p>This rule raises issues when the following .Net Core methods are called:
 <code>Microsoft.AspNetCore.Builder.IApplicationBuilder.UseDeveloperExceptionPage</code>,
-<code>Microsoft.AspNetCore.Builder.IApplicationBuilder.UseDatabaseErrorPage</code>. No Issue is raised when those calls are disabled by <code>if
```

Comment:
do we need to do a code change for `env.IsDevelopment()`?

Reply:
Indeed, this was not implemented yet. I will revert the changes.